web/src/engine/websites/Komiku.ts

```diff
@@ -0,0 +1,42 @@
+import { Tags } from '../Tags';
+import icon from './Komiku.webp';
+import { DecoratableMangaScraper, Manga, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchRequest } from './../FetchProvider';
+
+const paths = ['manga', 'manhua', 'manhwa'];
+
+function MangaLabelExtractor(element: HTMLElement) {
+    return element.textContent.replace('Komik', '').trim();
+}
+
+@Common.MangaCSS(/https:\/\/komiku\.id/, 'article header#Judul h1[itemprop="name"]', MangaLabelExtractor)
+@Common.ChaptersSinglePageCSS('table#Daftar_Chapter td.judulseries a')
+@Common.PagesSinglePageCSS('section#Baca_Komik img')
+@Common.ImageDirect()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('komiku', `Komiku`, 'https://komiku.id', Tags.Language.Indonesian, Tags.Media.Manga);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override async FetchMangas(provider: MangaPlugin): Promise<Manga[]> {
+        //need to get 3 categories since the "full list" is not full

```

Comment:
Comment seems redundant as the source code is quite clean and easy to understand